src/Gaufrette/Filesystem.php

```diff
@@ -344,4 +344,24 @@ private function isFileInRegister($key)
     {
         return array_key_exists($key, $this->fileRegister);
     }
+
+    /**
+     * Clear files register
+     */
+    public function clearCache()
```

Comment:
I think `clearCache` is more precice, cause this is cache actually. @wysow what do you think?